MIGRATION_GUIDE.md

```diff
@@ -6,6 +6,83 @@ across different versions.
 
 > [!TIP]
 > We highly recommend upgrading the versions one by one instead of bulk upgrades.
+ 
+## v0.98.0 ➞ v0.99.0
+
+### snowflake_task resource changes
+
+Breaking changes:
+- `config` field now can be defined without $ signs
+
+Before:
+```terraform
+resource "snowflake_task" "config_example" {
+  # ...
+  config = "$$${\"my_config_key\":\"my_config_value\"}$$"
```

Comment:
`$$$` or `$$`?

Reply:
It was `$$$` because `${` is treated as string interpolation in HCL and you have to escape it with `$${`